src/__init__.py

```diff
@@ -340,6 +340,10 @@
 
 __version__ = moduleInfo()['version']
 
+def print_err(s):
+    print(s, file=sys.stderr)
```

Comment:
Could you use `print(s, file=sys.stderr)` directly? It is not a big deal but in this case the wrapper is too 'thin' to deserve a separate function.